lib/omni_auth/strategies/email.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# Based on https://github.com/weg-li/weg-li/blob/master/app/lib/omni_auth/strategies/email.rb
+# Original author: https://github.com/phoet
+
+require 'base64'
+
+module OmniAuth
+  module Strategies
+    class Email
+      include OmniAuth::Strategy
+
+      def request_phase
+        redirect '/sessions/email'
+      end
+
+      def callback_phase
+        token = request.params['token']
+        fail!(:authenticity_error) if token.blank?
+
+        begin
+          decoded_token = EmailAuthToken.decode(token)
+        rescue StandardError => e
+          fail!(:authenticity_error, e)
```

Comment:
This is silence the actual error and nothing will get reported to app-signal. I recommend raising the errors in the beginning, so you can see what fails and silence it later on.